src/components/molecules/TasksEmptyState.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <v-container class="v-container">
+		<base-image
+			:img-src="image"
+			:img-height="imgHeight"
+			fill="var(--color-primary)"
+			role="presentation"
+		/>
+		<h2 class="h4 text-center mt-8">{{ title }}</h2>
+		<h3 class="h4 text-center mt-3">{{ subtitle }}</h3>
```

Comment:
as title and subtitle are not required, I would suggest to add a v-if (you would need to try this out, if it works with the default "" value)

Additionally: If we do not display a h1 element, then this needs to be changed accordingly

```suggestion
		<h1 v-if="title" class="h4 text-center mt-8">{{ title }}</h1>
		<h2 v-if="subtitle" class="h4 text-center mt-3">{{ subtitle }}</h2>
```

Reply:
I made the title required, but left subtitle optional in case we want to display a shorter message. And the h2 will only be shown if the subtitle is passed as a prop.